changelogs/unreleased/6269-sunjayBhatia-minor.md

```diff
@@ -0,0 +1,4 @@
+## Update examples for monitoring Contour and Envoy
+
+Updates the documentation and examples for deploying a monitoring stack (Prometheus and Grafana) to scrape metrics from Contour and Envoy.
```

Comment:
Maybe we could add a link here to the guide as well https://projectcontour.io/docs/main/guides/prometheus/ ?